### PR TITLE
CSS & HTML

### PR DESCRIPTION
**Pull request by edwardycl**

### CODE REVIEW

**Comment by edwardycl** [comment]
querySelector들이 전부 fail하고있다.. test 수정 필요

**Comment by skystar-p** [comment]
커버리지 보소

**Comment by tirr-c** [comment]
프론트엔드 커버리지 보고가 안 돼서...

**Comment by web-mighty-homu** [comment]
:umbrella: The latest upstream changes (presumably #75) made this pull request unmergeable. Please resolve the merge conflicts.

**Comment by tirr-c** [comment]
?! 커버리지 올랐다

**Comment by web-mighty-homu** [comment]
:umbrella: The latest upstream changes (presumably #76) made this pull request unmergeable. Please resolve the merge conflicts.

**Comment by skystar-p** [comment]
머지 컴플릭트 났다

**Comment by edwardycl** [comment]
바로 수정했음 머지 ㄱㄱㄱ

File: frontend/src/app/menu-bar.component.css [hunk 1]
```diff
@@ -0,0 +1,29 @@
+.clickable {
+  cursor: pointer;
+}
+.logo-header-large {
+  padding: 0 0 0 0;
```
Comment (tirr-c) [inline frontend/src/app/menu-bar.component.css]: `padding: 0;`

File: frontend/src/app/menu-bar.component.css [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  justify-content: center;
+  background-color: #FFFFFF;
```
Comment (tirr-c) [inline frontend/src/app/menu-bar.component.css]: `background-color: white;` 또는 `background-color: #fff;`

File: frontend/src/app/profile-view.component.css [hunk 3]
```diff
@@ -0,0 +1,4 @@
+.error-message {
+    margin: 0 0 10px 0;
```
Comment (tirr-c) [inline frontend/src/app/profile-view.component.css]: `margin: 0 0 10px;`

File: frontend/src/app/menu-bar.component.css (hunk 1 above)
Comment (edwardycl) [reply frontend/src/app/menu-bar.component.css]: 바뀔 수 있으니 우선 0 0 0 0으로 남겨둘게

File: frontend/src/app/menu-bar.component.css (hunk 2 above)
Comment (edwardycl) [reply frontend/src/app/menu-bar.component.css]: 이건 white로 수정할게

**Comment by tirr-c** [comment]
@web-mighty-homu r+

**Comment by web-mighty-homu** [comment]
:hourglass: Testing commit 2aa91bcd1060fc09361918c831e4a0dd59345bc1 with merge ca07f944bc7a0516ee4e3b42b1f4ea524fba13e2...

**Comment by web-mighty-homu** [comment]
:sunny: Test successful - [status-travis](https://travis-ci.org/swsnu/swpp17-team2/builds/311137794?utm_source=github_status&utm_medium=notification)
Approved by: tirr-c
Pushing ca07f944bc7a0516ee4e3b42b1f4ea524fba13e2 to master...

**Pull request merged**